Cargo.toml

```diff
@@ -27,5 +27,6 @@ members = [
     "mixnet/client",
     "mixnet/protocol",
     "mixnet/topology",
+    "mixnet-utils"
```

Comment:
How about having a `mixnet` crate on the root? because we’re gonna implement a lot of mixnet-specific logic soon. At my latest PR, I’ve deleted the previous `mixnet` crate, so you can recreate from scratch

Reply:
Let me merge my PRs now